.github/workflows/dependabot.yml

```diff
@@ -0,0 +1,21 @@
+name: dependabot
+on: pull_request
+
+permissions:
+  pull-requests: write
+
+jobs:
+  dependabot:
+    runs-on: ubuntu-latest
+    if: ${{ github.actor == 'dependabot[bot]' }}
+    steps:
+      - name: metadata
+        id: metadata
+        uses: dependabot/fetch-metadata@v1.1.1
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+      - name: approve
+        run: gh pr review --approve "$PR_URL"
```

Comment:
Does this actually work? Did you test this? If I understand correctly, it won't work. If it does, I'm slightly bothered because it means that someone can submit a pull request containing an action that approves the pull request.

Reply:
It works: ![image](https://user-images.githubusercontent.com/773636/155766709-d642021d-ecd2-43d8-a54d-f0229fc36060.png)